requirements/edx/custom.txt

```diff
@@ -22,7 +22,7 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.1#egg=xblock-g
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.4#egg=xblock-diagnostic-feedback==0.2.4
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@0.4.10#egg=xblock-group-project-v2==0.4.10
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
-git+https://github.com/edx-solutions/api-integration.git@v2.5.1#egg=api-integration==2.5.1
+git+https://github.com/edx-solutions/api-integration.git@vv2.5.2#egg=api-integration==2.5.2
```

Comment:
version should be `v2.5.2` not `vv2.5.2`

Reply:
yes it was typo